assets/styles16.css

```diff
@@ -0,0 +1,175 @@
+  src: url("./assets/fonts/barlow.ttf") format("truetype");
+  font-weight: normal;
+}
+@font-face {
+  font-family: "barlow_bold";
+  src: url("./assets/fonts/barlow_bold.ttf") format("truetype");
+  font-weight: bold;
+}
+body {
+  font-family: "barlow", sans-serif;
+  background-color: rgb(61, 139, 180);
+  padding: 9px;
+  margin: 0;
+  box-sizing: border-box;
+  height: 100vh;
+}
+.page_container {
+  display: grid;
+  gap: 9px;
+  grid-template-columns: 100px 1fr 150px;
+  grid-template-rows: auto 1fr auto;
+  grid-template-areas:
+    "header header header"
+    "left_sidebar main right_sidebar"
+    "left_sidebar footer footer";
+}
+.header {
+  background-color: rgb(199, 211, 217);
+  grid-area: header;
+  grid-column: 1/4;
+}
+.nav {
+  font-family: "barlow_bold", sans-serif;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 9px;
+  gap: 21px;
+}
+.nav_link {
+  color: rgb(61, 139, 180);
+  font-size: 29px;
+}
+.nav_link:hover {
+  color: red;
+}
+.left_sidebar {
+  background-color: rgb(199, 211, 217);
+  grid-area: left_sidebar;
+  display: flex;
+  flex-direction: column;
+  padding: 9px;
+  gap: 9px;
+}
+.left_link {
+  color: rgb(61, 139, 180);
+  font-size: 14px;
+  text-decoration: none;
+}
+.left_link:hover {
+  color: red;
+}
+.main_container {
+  background-color: rgb(199, 211, 217);
+  grid-area: main;
+  display: grid;
+  gap: 9px;
+  padding: 9px;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(4, 200px);
+}
+.kitten {
+  position: relative;
+}
+.kitten:hover .kitten__text_container {
+  opacity: 1;
+}
+.kitten1 {
+  grid-column: span 2;
+  grid-row: span 2;
+}
+.kitten__text_container {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  opacity: 0;
+  transition: opacity 0.3s;
+}
+.kitten__text {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  color: white;
+  background-color: rgba(0, 0, 0, 0.5);
+  font-size: 24px;
+  padding: 21px;
+  margin: 0;
+}
+.kitten5 {
+  grid-column: 3/4;
+  grid-row: 2/4;
+  height: 409px;
+}
+.kitten__img {
+  width: 100%;
+  height: 200px;
+  margin: 0;
+  object-fit: cover;
+}
+.img1 {
+  height: 409px;
+}
+.img5 {
+  height: 409px;
+}
+.right_sidebar {
+  background-color: rgb(199, 211, 217);
+  grid-area: right_sidebar;
+  display: flex;
+  flex-direction: column;
+  padding: 9px;
+  gap: 9px;
+  margin: 0;
+}
+.puppy {
+  position: relative;
+  height: auto;
+  width: 132px;
+}
+.puppy__img {
+  height: auto;
+  width: 132px;
+  margin: 0;
+  transition: opacity 0.3s ease;
+}
+.puppy__text-container {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 132px;
+  height: 100%;
+  background-color: rgba(255, 255, 255, 0.6);
+  color: black;
+  margin: 0;
+  opacity: 0;
+  transition: opacity 0.3s ease;
+  font-size: 21px;
+  font-weight: bold;
+}
+.puppy__text {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(255, 255, 255, 0.6);
```

Comment:
since you already set background to text_container, there is no need to set it to text.
Same to half of styles that are duplicated for . puppy__text-container and . puppy__text.

Take a closer look and remove all redundant styles